package.json

```diff
@@ -17,11 +17,11 @@
     "start-server-no-dir": "cross-env TRILIUM_SAFE_MODE=1 TRILIUM_ENV=dev TRILIUM_SYNC_SERVER_HOST=http://tsyncserver:4000 nodemon src/www.ts",
     "qstart-server": "npm run qswitch-server && TRILIUM_SAFE_MODE=1 TRILIUM_DATA_DIR=./data TRILIUM_ENV=dev TRILIUM_SYNC_SERVER_HOST=http://tsyncserver:4000 nodemon src/www.ts",
     "start-electron": "rimraf ./dist && tsc && ts-node ./bin/copy-dist.ts && cross-env TRILIUM_SAFE_MODE=1 TRILIUM_DATA_DIR=./data TRILIUM_SYNC_SERVER_HOST=http://tsyncserver:4000 TRILIUM_ENV=dev electron ./dist/electron.js --inspect=5858 .",
-    "start-electron-no-dir": "cross-env TRILIUM_SAFE_MODE=1 TRILIUM_ENV=dev TRILIUM_SYNC_SERVER_HOST=http://tsyncserver:4000 electron --inspect=5858 .",
```

Comment:
Whats the reason for removing `TRILIUM_SYNC_SERVER_HOST`

Reply:
@alexpietsch , I removed it in order to be able to test the sync.
More specifically, `start-electron-no-dir` (as opposed to `start-electron`) uses the "real" data directory, yet it uses a hardcoded sync server (`http://tsyncserver:4000`). Since we are using the real data directory, I don't see why we should have a hardcoded sync server, instead we should use the one configured in the server.

Honestly I don't even think `tsyncserver:4000` being hardcoded serves us well right now, since I'm not sure how it's set up. I suppose it's just a server instance hosted locally and added to `/etc/hosts`.

Reply:
@alexpietsch ,

We can revert the change later if needed, however I can't keep this blocked any longer since sync is impossible for me otherwise.